docs/how_to_add_packages.md

```diff
@@ -45,6 +45,14 @@ This process helps conan-center-index against spam and malicious code. The proce
 When submitting a pull request for the first time, you will be prompted to sign the [CLA](CONTRIBUTOR_LICENSE_AGREEMENT.md) for your code contributions.
 You can view your signed CLA's by going to <https://cla-assistant.io/> and signing in.
 
+
+## Inactivity and user removal
+
+For security reasons related to the CI, when an user no longer contributes for a longe period, it will consired inactive and removed from the authorized users list.
```

Comment:
```suggestion
For security reasons related to the CI, when a user no longer contributes for a long period, it will be considered inactive and removed from the authorized user's list.
```